source/extensions/filters/http/lua/lua_filter.cc

```diff
@@ -428,6 +435,129 @@ int StreamHandleWrapper::luaLogCritical(lua_State* state) {
   return 0;
 }
 
+const EVP_MD* StreamHandleWrapper::getDigest(const absl::string_view& name) {
+  auto hash = absl::AsciiStrToLower(name);
```

Comment:
(throughout whole PR) only use `auto` when the type is obvious (e.g. casting, new)